packages/extension-agent/src/computer/backends/e2b.ts

```
@@ -465,11 +496,45 @@ export class E2BComputerSession implements ComputerSessionApi {
+        sandbox?: SandboxWrapper
+    ) {
+        const current = sandbox ?? (await this.ensureSandbox())
+
+        try {
+            return await current.commands.run(command, options)
+        } finally {
+            try {
+                await current.setTimeout(this.cfg.timeoutMs)
+            } catch (err) {
+                if (!isMissingSandboxError(err)) {
+                    throw err
```

Biome (2.4.7)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)